src/Tribe/Admin/Notice/Marketing.php

```diff
@@ -0,0 +1,53 @@
+<?php
+/**
+ * @internal This class may be removed or changed without notice
+ */
+class Tribe__Events__Admin__Notice__Marketing {
+	/**
+	 * Register marketing notices.
+	 *
+	 * @since TBD
+	 */
+	public function hook() {
+		tribe_notice(
+			'tribe-events-upcoming-survey',
+			array( $this, 'notice' ),
+			array(
+				'dismiss' => 1,
+				'type'    => 'info',
+				'wrap'    => 'p',
+			),
+			array( $this, 'should_display' )
+		);
+	}
+
+	/**
+	 * @since TBD
+	 *
+	 * @return bool
+	 */
+	public function should_display() {
+		return tribe( 'admin.helpers' )->is_screen()
+			&& date_create()->format( 'Y-m-d' ) < '2018-06-08';
```

Comment:
Are you sure this is PHP 5.2 compat?

Reply:
Yep, should be good: `date_create()` was introduced in 5.2.0 and `DateTime::format()` was introduced in 5.2.1, so within bounds (min supported version for us being 5.2.4).

Reply:
I meant the usage o `date_create` without variable assign, into a `->` usage.